src/pages/Home.tsx

```diff
@@ -42,24 +45,33 @@ export const Home = () => {
     }
   }, [token]);
 
+  console.log(tokenStore.statusResult);
+  console.log(ballot.ballotLoading);
+
   return (
     <div>
-      <IntroSection/>
-      <ContentSection/>
-      {ballot.ballot && !ballot.ballotError && !vote.voteResult && (
-        <VotingSection
-          onTokenReceive={handleCaptchaTokenReceive}
-          ballot={ballot}
-          onVote={handleVote}
-        />
-      )}
-      {ballot.ballotError && !vote.voteResult && (
+      <IntroSection />
+      <ContentSection />
+      {ballot.ballot &&
+        !ballot.ballotError &&
+        !vote.voteResult &&
+        tokenStore.statusResult === "VALID" && (
+          <VotingSection
+            onTokenReceive={handleCaptchaTokenReceive}
+            ballot={ballot}
+            onVote={handleVote}
+          />
+        )}
+      {(ballot.ballotLoading || (ballot.ballotError && !vote.voteResult)) && (
         <NoVotingSection
-          type={(ballot.ballotError.message ?? "") as BallotError}
+          type={(ballot.ballotError?.message ?? "LOADING") as NoVotingType}
         />
       )}
-      {vote.voteResult && (
-        <CouchPanel mainText={"Du hast abgestimmt :)"} subText={"Dann entspann dich doch!"}/>
+      {tokenStore.statusResult != ("VALID" && "MISSING") && (
```

Comment:
This is not how JavaScript works. `"VALID" && "MISSING"` evaluates to `"MISSING"`, because `"VALID"` is [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy). I think you meant `tokenStore.statusResult !== "VALID" && tokenStore.statusResult !== "MISSING"`. Or a more functional solution: `!["VALID", "MISSING"].includes(tokenStore.statusResult)`